frontend/libs/studio-components/src/components/StudioStatusRadioGroup/StudioStatusRadioGroup.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { type ChangeEvent, type ReactElement } from 'react';
+import classes from './StudioStatusRadioGroup.module.css';
+import cn from 'classnames';
+import { StudioLabelAsParagraph } from '../StudioLabelAsParagraph';
+import { StudioParagraph } from '../StudioParagraph';
+
+type StudioStatusRadioButtonColor = 'success' | 'info';
```

Comment:
I noticed in the video in the PR description that there was a 3rd color (yellow - warning?), but that is not available here any more. Is that intentional?

Reply:
Yes, agreed with @Annikenkbrathen after PR was created to remove it. Thanks for checking 😄